lib/utils/location.js

```diff
@@ -45,7 +45,7 @@ export function matchesPageLocation(includes, excludes) {
 /* eslint-disable key-spacing */
 export const regexes = {
 	frontpage:        /^\/(?:hot|new|rising|controversial|top)?(?:\/|$)/i,
-	comments:         /^\/(?:r\/([\w\.]+)\/)?comments(?:\/|$)/i,
+	comments:         /^\/(?:r\/([\w\.]+)\/)?comments(?:\/([a-z0-9]+)\/\w*)?(?:\/|$)/i,
```

Comment:
Why remove the `(?:\/|$)`? Are there any valid urls where the comment id or `comments` isn't followed by a slash or the end of the pathname? (it avoids matching a path section like `/comments_something_something`, though that may not be relevant here)

The comment id is optional in the current regex so that it matches `/r/enhancement/comments` (legacy behaviour). We could change that, but I'm not sure what a better page type would be and I don't really want to add a new one.